README.md

```diff
@@ -163,4 +171,4 @@ See the [Configuration] document for information on the options.
 
 ## Running
 
-Run [`wayfire`][Manual] from a TTY, or via a Wayland-compatible login manager.
+Run [`wayfire`][Manual] from a TTY, or via a Wayland-compatible login manager.
```

Comment:
What changed here?

Reply:
In fact, I don’t know why this happens, vim automatically modifies it

After I manually added a newline, the diff was normal